README.md

```diff
@@ -1 +1,196 @@
 # Event-triggered Notifications Tutorial
+
+An event-triggered notification is an automated messages sent in response to an event.
+
+There are three approaches for implementing these types of notifications:
+
+- Leverage **Segments** for sending messages.
+- Use the OneSignal Notification API.
+- Use a platform SDK.
+
+This tutorial series will cover the three methods outlined above, but first, I need to cover the building blocks that make this possible: events, segments, and automated messages.
+
+## Example Use-case
```

Comment:
I suggest swapping this and the next paragraph as it forms a more direct line of increasing depth of subject matter coverage throughout the document.  i.e. pyramid-shaped